Integrations/Panorama/Panorama.yml

```diff
@@ -2232,8 +2232,8 @@ script:
       required: false
       secret: false
     - default: false
-      description: The Search time for the request. For further information, see the
-        Panorama XML API Documentation.
+      description: 'The Search time for the request.  e.g: "2019/12/26 00:00:00",
```

Comment:
For example: "2019/12/26 00:00:00", "2020/01/10". For more information, see the Panorama XML API documentation.'